kasa/iot/modules/dimmer.py

```diff
@@ -0,0 +1,245 @@
+"""Implementation of the dimmer config module found in dimmers."""
+
+from __future__ import annotations
+
+import logging
+
+from ...exceptions import KasaException
+from ...feature import Feature
+from ..iotmodule import IotModule, merge
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class Dimmer(IotModule):
+    """Implements the dimmer config module."""
+
+    def _initialize_features(self) -> None:
+        """Initialize features after the initial update."""
+        # Only add features if the device supports the module
+        if "get_dimmer_parameters" not in self.data:
+            return
+
+        self._add_feature(
+            Feature(
+                device=self._device,
+                container=self,
+                id="dimmer_threshold_min",
+                name="Minimum Dimming Level",
```

Comment:
```suggestion
                name="Minimum dimming level",
```
I don't think we are using title casing currently anywhere, so only the first letter should be capitalized.